### PR TITLE
Removed extra RETRY_DELAY definition.

### PR DESCRIPTION
**Pull request by ebadiere**
Cleanup.  Related issue, #571 

**Related issue(s)**:

Fixes #

### CODE REVIEW

File: local.env [hunk 1]
```diff
@@ -14,7 +14,6 @@ PRIVATE_KEYS=
 0x95eac372e0f0df3b43740fa780e62458b2d2cc32d6a440877f1cc2a9ad0c35cc,
 0x6c6e6727b40c8d4b616ab0d26af357af09337299f09c66704146e14236972106"
 
-RETRY_DELAY=3000 # ms
```
Comment (Nana-EC) [inline local.env:17]: SO do we still need this large polling logic or can this be reduced by default?

Comment (ebadiere) [reply local.env:17]: We do, we just don't need it defined twice.

**Comment by Nana-EC** [comment]
@ebadiere please rebase

**Comment by ebadiere** [comment]
Not needed.

**Pull request closed without merging**